ETH-Merge-Bots/Milestones-in-TTD/src/agent.ts

```diff
@@ -0,0 +1,122 @@
+import { BlockEvent, Finding, HandleBlock } from "forta-agent";
+import {
+  getNumberOfDays,
+  getAverageBlockTime,
+  getEstimatedNumberOfBlocksUntilMerge,
+  getAvgBlockDifficulty,
+} from "./utils";
+import { TERMINAL_TOTAL_DIFFICULTY, MILESTONES, ETH_BLOCK_DATA } from "./eth.config";
+import { createFinding, createFinalFinding } from "./findings";
+import BigNumber from "bignumber.js";
+
+let blockCounter: number = 0;
+let firstTimestamp: number = 0;
+
+const avgBlockDifficulties: BigNumber[] = [];
```

Comment:
```suggestion
const blockDifficulties: BigNumber[] = [];
```

Too much of a detail, but this is more correct, right?

Reply:
that's right 👍